_docs-v5/event-source/events-json-feed.md

```diff
@@ -208,13 +208,13 @@ and
   [
     {
       "title": "Event 1",
-      "start": 1567674000,
-      "end": 1567706400
+      "start": 1567674000000, // please specify dates in unix format in milliseconds, not seconds
```

Comment:
just make the comment `please specify dates in milliseconds, not seconds`

sometimes people call second-based times "unix timestamps", which is the opposite of what we want to communicate, so let's just avoid the confusion altogether

Reply:
@arshaw done